doc/radosgw/multisite.rst

```diff
@@ -64,6 +64,12 @@ something like this:
 .. image:: ../images/zone-sync2.png
    :align: center
 
+At the top of this diagram, we see two applications (also known as "clients"). The application on the right is both writing and reading data from the Ceph Cluster, by means of the RADOS Gateway (RGW).
+
+In the middle of this diagram, we see two zones, each of which contains an instance of a RADOS Gateway (RGW). These instances of RGW are handling the movement of data from the applications (also known as "clients") to the zonegroup. The arrow from the master zone (US-EAST) to the secondary zone (US-WEST) represents an act of data synchronization.
```

Comment:
Leave out `(also known as "clients")` after the first instance

Reply:
I've removed the later instance.